e2e-tests/contentful/gatsby-config.js

```diff
@@ -0,0 +1,20 @@
+const path = require(`path`)
+
+module.exports = {
+  siteMetadata: {
+    title: `Gatsby Contentful e2e`,
+  },
+  plugins: [
+    {
+      resolve: `gatsby-source-contentful`,
+      options: {
+        spaceId: `k8iqpp6u0ior`,
+        accessToken: `hO_7N0bLaCJFbu5nL3QVekwNeB_TNtg6tOCB_9qzKUw`,
```

Comment:
I assume this is read-only?

Reply:
yes it is and a space only for this porpuse